src/pages/permissions/PermissionGroups.tsx

```diff
@@ -0,0 +1,307 @@
+  const panelParams = usePanelParams();
+  const [search, setSearch] = useState("");
+  const [selectedGroupNames, setSelectedGroupNames] = useState<string[]>([]);
+
+  if (error) {
+    notify.failure("Loading groups failed", error);
+  }
+
+  useEffect(() => {
+    if (panelParams.group) {
+      setSelectedGroupNames([panelParams.group]);
+    }
+  }, [panelParams.group]);
+
+  const headers = [
+    { content: "Name", className: "name", sortKey: "name" },
+    {
+      content: "Description",
+      className: "description",
+      sortKey: "description",
+    },
+    {
+      content: "Identities",
+      sortKey: "identities",
+      className: "u-align--right identities",
+    },
+    {
+      content: "Permissions",
+      sortKey: "permissions",
+      className: "u-align--right permissions",
+    },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const filteredGroups = groups.filter(
+    (group) =>
+      !search ||
+      group.name.toLowerCase().includes(search) ||
+      group.description.toLowerCase().includes(search),
+  );
+
+  const selectedGroups = groups.filter((group) =>
+    selectedGroupNames.includes(group.name),
+  );
+
+  const rows = filteredGroups.map((group) => {
+    const allIdentityIds = getIdentityIdsForGroup(group);
+    return {
+      name: group.name,
+      className: "u-row",
+      columns: [
+        {
+          content: group.name,
+          role: "cell",
+          "aria-label": "Name",
+          className: "u-truncate name",
+          title: group.name,
+        },
+        {
+          content: <span>{group.description}</span>,
+          role: "cell",
+          "aria-label": "Description",
+          className: "description",
+          title: group.description,
+        },
+        {
+          content: allIdentityIds.length,
+          role: "cell",
+          className: "u-align--right identities",
+          "aria-label": "Identities in this group",
+        },
+        {
+          content: group.permissions?.length || 0,
+          role: "cell",
+          className: "u-align--right permissions",
+          "aria-label": "Permissions for this group",
+        },
+        {
+          className: "actions u-align--right",
+          content: <GroupActions group={group} />,
+          role: "cell",
+          "aria-label": "Actions",
+        },
+      ],
+      sortData: {
+        name: group.name.toLowerCase(),
+        description: group.description.toLowerCase(),
+        permissions: group.permissions?.length || 0,
+        identities: allIdentityIds.length,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({
+    rows,
+    defaultSort: "name",
+  });
+
+  if (isLoading) {
+    return <Loader text="Loading groups" />;
+  }
+
+  const getTablePaginationDescription = () => {
+    if (panelParams.panel) {
+      return `Showing selected ${pluralize("group", sortedRows.length)}`;
+    }
```

Comment:
This should be removed now. Maybe we don't need the whole thing here and can rely on the default?
```suggestion
```

Reply:
Aaah yes these will need to change, same in identities list. The default from `SelectedTableNotification` should be fine, but it will need to be adjusted so that we can hide the select/clear actions if the panel is open.